src/models/program.js

```diff
@@ -1,34 +1,35 @@
-const {
-  Model,
-} = require('sequelize');
-
-export default (sequelize, DataTypes) => {
-  class Program extends Model {
-    static associate(models) {
-      Program.belongsTo(models.Grant, { foreignKey: 'grantId', as: 'grant' });
-    }
-  }
-  Program.init({
-    id: {
-      type: DataTypes.INTEGER,
-      allowNull: false,
-      defaultValue: null,
-      primaryKey: true,
-      autoIncrement: false,
-    },
-    grantId: {
-      type: DataTypes.INTEGER,
-      allowNull: false,
-    },
-    programType: DataTypes.STRING,
-    startYear: DataTypes.STRING,
-    startDate: DataTypes.STRING,
-    endDate: DataTypes.STRING,
-    status: DataTypes.STRING,
-    name: DataTypes.STRING,
-  }, {
-    sequelize,
-    modelName: 'Program',
-  });
-  return Program;
-};
+const {
+  Model,
+} = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class Program extends Model {
+    static associate(models) {
+      Program.belongsTo(models.Grant, { foreignKey: 'grantId', as: 'grant' });
+      Program.hasMany(models.ProgramPersonnel, { foreignKey: 'programId', as: 'programPersonnel' });
+    }
+  }
+  Program.init({
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      defaultValue: null,
+      primaryKey: true,
+      autoIncrement: false,
+    },
+    grantId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    programType: DataTypes.STRING,
+    startYear: DataTypes.STRING,
+    startDate: DataTypes.STRING,
+    endDate: DataTypes.STRING,

```

Comment:
Can these be some form of date column

Reply:
I think it comes in as a string.

Reply:
I'm just realizing these are not part of this PR.